internal/hooks/NewRound_DoCombat.go

```diff
@@ -1075,18 +1075,15 @@ func handleAffected(affectedPlayerIds []int, affectedMobInstanceIds []int) {
 		if user := users.GetByUserId(userId); user != nil {
 
 			if user.Character.Health <= -10 {
+
 				user.Command(`suicide`) // suicide drops all money/items and transports to land of the dead.
+
 			} else if user.Character.Health < 1 {
 
-				user.SendText(`<ansi fg="red">you drop to the ground!</ansi>`)
-
-				if room := rooms.LoadRoom(user.Character.RoomId); room != nil {
-					room.SendText(
-						fmt.Sprintf(`<ansi fg="username">%s</ansi> <ansi fg="red">drops to the ground!</ansi>`, user.Character.Name),
-						user.UserId)
-				}
+				events.AddToQueue(events.PlayerDrop{UserId: user.UserId, RoomId: user.Character.RoomId})
```

Comment:
Just a little confused, using `user.Character.Health` to distinguish between `suicide` or `PlayerDrop`, what's the reason?

Reply:
the suicide command is invoked when the player dies (hits -10 health).

Between 0 and -10, they drop to the ground and bleed out slowly.

Reply:
This is ancient logic, and may be replaced eventually - maybe with some sort of attribute or something, but works well enough for now :)

Once they drop under 0 hp, combat no longer can target them. They "drop to the ground" and bleed hp every auto-healing round that passes.

Once they hit -10 hp, they force-initiate a `suicide` which for players, is just a death.